src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs

```diff
@@ -103,8 +103,9 @@ public bool IsAnimated
 		/// Backing BindableProperty for the <see cref="BackgroundColor"/> property.
 		/// </summary>
 		public static new BindableProperty BackgroundColorProperty =
-			BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(BadgeView), defaultValue: Color.Default);
-
+			BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(BadgeView), defaultValue: Color.Default,
+				propertyChanged: OnLayoutPropertyChanged);
```

Comment:
do we need to invalidate the entire layout?

Reply:
would be better to update only the necessary part, but it's common practice for this View (other props do the same)

Reply:
I see we should back this at some point as an improvement, in another issue/PR of course. Let's ignore this right now